app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,60 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = Recipe.where(user: current_user)
```

Comment:
Hi @FrederickMih, thank you for your feedback but I was confused as in the requirements it says:

Recipes list:
- Should display a list of recipes **created by the logged-in user** as in the wireframe.

I wasn't sure of whether to display all recipes or not and it would be time-consuming to refactor code due to time issues. I'll surely keep that suggestion in mind and will come back later to make it functional in that way.